examples/tutorials/08_research_tutorial/2_Research_advanced.ipynb

```diff
@@ -0,0 +1,1030 @@
+{
```

Comment:
<p>Explicitly explain that testing is being performed with the train dataset</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/337/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>